reactioncommerce/reaction
# Fixes #3692: bulk order status corrected

Resolves #3692  
Impact: minor  
Type: bugfix

## Issue
The issue is caused because of the way the state was saved. It only stored information about the latest status of shipping.

## Solution
The state is modified to store the status of all values of shipping.
Also fixed some loader related issue when jumping workflow

## Testing
1. Create 2+ orders
1. Use the bulk order editor to move the orders forward in their workflow (i.e. from Picked to Packed).
1. See that the checkmark on the previous state is still there.
1. Try jumping the status to Shipped. Observe the loader on the states that you jumped.
1. Regress back to packed state. Observe the loader on the states you jumped.
